app/views/homepage/index.html.erb

```diff
@@ -109,29 +109,21 @@
 
         <div class="govuk-grid-column-one-third">
           <a href="/transition" aria-hidden="true" tabindex="-1">
-            <%= image_tag "homepage/time-is-running-out--480w.jpg", {
+            <%= image_tag "homepage/transition-period.png", {
```

Comment:
Note from our discussions: this is still in the repo despite not being used (see [here](https://github.com/alphagov/frontend/blob/master/app/assets/images/homepage/transition-period.png)); we agreed to stick with the single variant, noting it's only ~5KB in size!